cypress/components/alert/alert.cy.tsx

```diff
@@ -133,7 +133,7 @@ context("Testing Alert component", () => {
       closeIconButton()
         .click()
         .then(() => {
-          // eslint-disable-next-line no-unused-expressions
+          // eslint-disable-next-line no-unused-expressions, jest/valid-expect
```

Comment:
We should be ok to disable the jest linting rules here. Ideally, eslint shouldn't be linting the cypress tests with jest rules, so we will need to address this for you as part of FE-5758.